src/main/java/controller/BaseballManagement.java

```diff
@@ -0,0 +1,98 @@
+package controller;
+
+import model.Computer;
+import view.InputView;
+import view.OutputView;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+import static model.Computer.LIMIT_NUMBER;
+import static model.GameFlag.*;
+
+public class BaseballManagement {
+    private InputView inputView;
+    private OutputView outputView;
+
+    public BaseballManagement() {
+        inputView = new InputView();
+        outputView = new OutputView();
+    }
+
+    public void GameStart() {
+        while (ReGameFlag == GAME_START) {
+            run();
+            CheckReGameFlag(Integer.parseInt(inputView.inputMenu()));
+            CheckFinishGameFlag(true);
+        }
+    }
+
+    private void run() {
+        Computer computer = new Computer();
+        List<Integer> userBaseballNumber = new ArrayList<>();
+        System.out.println(computer.getBaseballNum().get(0)+"" + computer.getBaseballNum().get(1) + "" +computer.getBaseballNum().get(2));
```

Comment:
```suggestion
        System.out.println(computer.getBaseballNumber());
```

이렇게 string 값을 가져오라는 메소드를 부르면 어떨까요?? :)
그리고 출력문이 컨트롤러에 있는게 안어울리는 것 같아요!

```
    // Computer.java
    public String getBaseballNumber() {
        // Computer ball String 값 리턴
    }
```